src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -375,6 +375,11 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('autoReveal', "Controls whether the explorer should automatically reveal and select files when opening them."),
 			'default': true
 		},
+		'explorer.autoRevealNoScroll': {
+			'type': 'boolean',
+			'description': nls.localize('autoRevealNoScroll', "Controls whether the explorer should not automatically scroll to revealed files."),
```

Comment:
Please chagne to "should automatically" the word "not" makes the sentence more confusing